sdc/crypto/scripts/generate_secrets.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```

Comment:
We probably don't need this any more given that it isn't really a script anymore?